src/test/java/io/cdap/plugin/salesforce/plugin/source/batch/SalesforceInputFormatProviderTest.java

```diff
@@ -0,0 +1,146 @@
+        assertEquals("S Object Name Field", salesforceMultiSourceConfig.getSObjectNameField());
+        assertEquals("username", salesforceMultiSourceConfig.getUsername());
+        assertEquals("https://example.org/example", salesforceMultiSourceConfig.getLoginUrl());
+        Set<String> whiteList = salesforceMultiSourceConfig.getWhiteList();
+        assertEquals(1, whiteList.size());
+        assertTrue(whiteList.contains("White List"));
+        AuthenticatorCredentials authenticatorCredentials = salesforceMultiSourceConfig.getAuthenticatorCredentials();
+        assertNull(authenticatorCredentials.getConsumerKey());
+        assertEquals("PluginProperties{properties={}, macros=Macros{lookupProperties=[], macroFunctions=[]}}",
+                rawProperties.toString());
+        assertNull(authenticatorCredentials.getConsumerSecret());
+        assertSame(oAuthInfo1, authenticatorCredentials.getOAuthInfo());
+        assertNull(authenticatorCredentials.getUsername());
+        assertEquals("PluginProperties{properties={}, macros=Macros{lookupProperties=[], macroFunctions=[]}}",
+                properties.toString());
+        assertEquals("https://example.org/example", oAuthInfo1.getInstanceURL());
+        assertNull(authenticatorCredentials.getLoginUrl());
+        assertNull(authenticatorCredentials.getPassword());
+        assertEquals("token", oAuthInfo1.getAccessToken());
+        Map<String, String> props = properties.getProperties();
+        assertTrue(props.isEmpty());
+        Macros macros = rawProperties.getMacros();
+        assertTrue(macros.getLookups().isEmpty());
+        assertTrue(macros.getMacroFunctions().isEmpty());
+        assertEquals("Macros{lookupProperties=[], macroFunctions=[]}", macros.toString());
+        assertTrue(stringStringMap.isEmpty());
+        assertTrue(salesforceSplitList.isEmpty());
+    }
+
+    @Test
+    public void testInputFormatProviderWithOuathInfoAsNull() {
+        HashMap<String, String> stringStringMap = new HashMap<>(1);
+        List<SalesforceSplit> salesforceSplitList = new ArrayList<>();
+        salesforceMultiSourceConfig = new SalesforceMultiSourceConfig("Reference Name",
+                "Consumer Key", "Consumer Secret", "username", "password", "https://example.org/example", "2020-03-01",
+                "2020-03-01", "Duration", "Offset", "White List", "Black List", "S Object Name Field", "token", null);
+
+        SalesforceInputFormatProvider actualSalesforceInputFormatProvider = new SalesforceInputFormatProvider(
+                salesforceMultiSourceConfig, stringStringMap, salesforceSplitList, "S Object Name Field");
+    }
+
+
+    @Test
+    public void testCInputFormatConfigurationSize() {
+        HashMap<String, String> schemas = new HashMap<>(1);
+
+        SalesforceInputFormatProvider salesforceInputFormatProvider = new SalesforceInputFormatProvider
+                (salesforceMultiSourceConfig, schemas, new ArrayList(),
```

Comment:
new HashMap<>()

Reply:
Removed the size/capacity argument from `new HashMap<>()`